src/Streamly/Internal/Data/Array/Stream/Mut/Foreign.hs

```diff
@@ -123,6 +135,13 @@ packArraysChunksOf n (D.Stream step state) =
 
     step' _ (SpliceYielding arr next) = return $ D.Yield arr next
 
+-- | "packArraysChunksOf" buy wrapped as a SerialT stream
+{-# INLINE_NORMAL packArraysChunksOf #-}
+packArraysChunksOf :: (MonadIO m, Storable a)
+    => Int -> SerialT m (Array a) -> SerialT m (Array a)
+packArraysChunksOf n (SerialT xs) =
+    SerialT $ D.toStreamK $ packArraysChunksOfD n (D.fromStreamK xs)
+
```

Comment:
What is the motivation for this API. Is this APIs used or planned to be used somewhere?